LiipImagineBundle.php

```diff
@@ -67,5 +70,9 @@ public function build(ContainerBuilder $container): void
         $extension->addLoaderFactory(new FileSystemLoaderFactory());
         $extension->addLoaderFactory(new FlysystemLoaderFactory());
         $extension->addLoaderFactory(new ChainLoaderFactory());
+
+        $container->registerForAutoconfiguration(LoaderLoaderInterface::class)->addTag('liip_imagine.filter.loader');
```

Comment:
this is only available since [5.3](https://symfony.com/blog/new-in-symfony-5-3-service-autoconfiguration-and-attributes) - can you check the symfony version before calling it?

the 5.4 build and lowest version builds fail - not sure if they install a too old symfony or if there is something else as well that we need to do for the test to work on legacy versions.

Reply:
i'd also be ok to only do it with symfony 6 or 7 and otherwise skip.

the doc should mention from what symfony version on this works.

Reply:
I did update the code to check if the function exists and updated the doc too.

Reply:
Well actually it looks like this was introduced a while ago looking at this commit https://github.com/symfony/symfony/commit/18627bf9f658ef4c986d93193cb01af5478bf3a3

But here it is returning `null` which is rather strange as the first implementation always return a `Symfony\Component\DependencyInjection\ChildDefinition`

Reply:
Oh okay, this is because we are using a MockObject in place of a real container in the test.

Reply:
I did revert my changes and add the method to the mock which should fix the tests.